prow/config.yaml

```diff
@@ -62,18 +62,17 @@ presubmits:
     - master
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:0.11
+      - image: gcr.io/k8s-testimages/bazelbuild:v20170720-7762904d
         args:
-        - "--pull=$(PULL_REFS)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
Periodic has a diff pattern from presubmit and postsubmit jobs. 

The env var pattern encourages/tempts developers into adopting bad practices by relying on implicit dependencies and global state. We should replace this with explicit, local config wherever possible. 

PodPresets will eliminate the boilerplate.